app/lib/fields/patternfly_form_builder.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
 class Fields::PatternflyFormBuilder < Fields::FormBuilder
-  def output_html(field, options = {})
-    input_type = field.choices.present? ? :patternfly_select : :patternfly_input
```

Comment:
Well, the problem is that we can only get the correct input type after this block: https://github.com/3scale/porta/blob/fix-country-field-on-account-edit/app/lib/fields/form_builder.rb#L36-L42

We can't get it from the original `field` argument passed to `output_html`.

Otherwise, we'd need to split this `output_html` method in two parts and override just one part.

Reply:
I've moved some things around... How does it look now?
As for me - it still looks pretty confusing,but that's just the original implementation is quite confusing also :upside_down_face: - too many `field` variables and `options` variables